code/modules/admin/admin_verbs.dm

```diff
@@ -794,6 +799,20 @@ var/list/admin_verbs_cciaa = list(
 			verbs |= /client/proc/readmin_self
 	feedback_add_details("admin_verb","DAS") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/toggle_aooc()
+	set name = "Toggle AOOC"
+	set category = "Admin"
+	if(!check_rights(R_ADMIN|R_MOD|R_CCIAA))
+		return
+	if(holder)
+		if (toggleantagooc() == FALSE)
+			to_chat(src, "<span class='interface'>AOOC is now muted</span>")
```

Comment:
Spacing here, please. Keep stuff readable if you can.

Reply:
What's the spacing issue? Looks fine to me really.